docs/user/automation-rules.rst

```diff
@@ -82,11 +89,8 @@ Currently, the following actions are available:
 Order
 -----
 
-The order your rules are listed in  :guilabel:`Admin` > :guilabel:`Automation Rules` matters.
-Each action will be performed in that order,
-so first rules have a higher priority.
-
-You can change the order using the up and down arrow buttons.
+When a new webhook event is received from your Git provider,
+All rules with a successful match will have their action triggered in the order they appear on the :guilabel:`Automation Rules` page.
```

Comment:
I need some feedback here. In a former section, we state:

> When a rule matches a new version, (...)

We should be more specific about the trigger. What happens when a new rule is added for a tag or branch and then _something new_ shows up? What happens when a Git tag is deleted and re-added?